starfyre/parser.py

```diff
@@ -20,9 +21,20 @@ class RootParser(HTMLParser):
     # this is the grammar for the parser
     # we need cover all the grammar rules
 
-    generic_tags = ["div", "p", "b", "span", "i", "button"]
-
-    def __init__(self, component_local_variables, component_global_variables, css, js):
+    generic_tags = [
```

Comment:
One suggestion - Can we convert it to a set instead of a list?

Reply:
Sure, I will look into this.